### PR TITLE
[Site Performance] Image optimization

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabh1908@gmail.com>

**Description**
1. Removed duplicate images in various folders. Imported images from /assets folder instead.
2. Recommending the use of `<StaticImages/>` instead of `` to improve image optimization and site performance.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Abhijay007** [comment]
@debo19 this PR is related to [this](https://github.com/layer5io/layer5/issues/2172) issue, right?

**Comment by debo19** [comment]
@Abhijay007 it's a part of this, yes.

**Comment by l5io** [comment]
🚀 Preview for commit 0fc108f746cc222211426947de70ab6402fc0349 at: https://61ec8f810b7692bcda055a69--layer5ng.netlify.app

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍

File: src/components/SocialLinks-Color/index.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const SocialLinksColor = () => {
           >
             <img height="30px" src={mail_icon} alt="mail" />
           </a>
-          <a href="http://slack.layer5.io/" target="_blank" rel="noreferrer">
+          <a href="https://slack.layer5.io/" target="_blank" rel="noreferrer">
```
Comment (leecalcote) [inline src/components/SocialLinks-Color/index.js:33]:
```suggestion
          <a href="https://slack.layer5.io/" target="_blank">
```

Let's make a point of leaving out  `rel="noreferrer"` on links within the layer5.io domain.

File: src/components/SocialLinks/index.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ const SocialLinks = () => {
             <img height="40 px" src={mail_icon} alt="mail" />
           </a>
           <a
-            href="http://slack.layer5.io/" 
+            href="https://slack.layer5.io/" 
             target="_blank"
             rel="noreferrer"
```
Comment (leecalcote) [inline src/components/SocialLinks/index.js:27]:
```suggestion
```

File: src/sections/Community/Handbook/projects.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ import meshery from "../../../assets/images/meshery/icon-only/meshery-logo-light
 import layer5icon from "../../../assets/images/layer5/5 icon/svg/light/5-light-bg.svg";
 import hawkIcon from "../../../assets/images/nighthawk/icon-only/SVG/nighthawk-logo.svg";
 import meshmateicon from "../../../assets/images/meshmate/meshmate-icon.svg";
+import PatternsLogo from "../../../assets/images/service-mesh-patterns/service-mesh-pattern.svg";
```
Comment (leecalcote) [inline src/sections/Community/Handbook/projects.js:7]: From PNG to SVG and from duplicate to centralized. Nice work.

**Review by leecalcote — APPROVED** [review]
`img` to `StaticImage` ✔️

**Comment by l5io** [comment]
🚀 Preview for commit 62bbb7896173714d569c68717fade8c5ffe05ada at: https://61ec9b120b7692c7400554d0--layer5ng.netlify.app

File: gatsby-config.js [hunk 4]
```diff
@@ -13,6 +13,9 @@ module.exports = {
   plugins: [
     "gatsby-plugin-sitemap",
     "gatsby-plugin-react-helmet",
+    "gatsby-plugin-image",
```
Comment (Nikhil-Ladha) [inline gatsby-config.js:16]: These are already used, check line 367.
Please undo this

Comment (Nikhil-Ladha) [reply gatsby-config.js:16]: All 3 of them

File: src/sections/Meshery/meshery-operator/index.js [hunk 5]
```diff
@@ -21,7 +21,7 @@ const MesheryOperatorPage = () => {
             <div className="hero-section">
               <Row>
                 <Col lg={6} md={6} sm={12}>
-                  <img src={Operator} className="logo" />
+                <StaticImage src="../../../assets/images/meshery-operator/meshery-operator.png" className="logo" />
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/meshery-operator/index.js:24]: One change, in order to keep the code clean.

After all the imports,  assign the image path which is going be used in the static image component to a variable like
`const Operator="enter the image path here"` and then use this variable name as the `src` value in `StaticImage` component.

Comment (Nikhil-Ladha) [reply src/sections/Meshery/meshery-operator/index.js:24]: Make this update at other places as well in the PR, where you are using StaticImage.

**Review by Nikhil-Ladha — COMMENTED** [review]
Also, have you removed all the images which are unused and is replaced by the generic image?

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Few changes are needed as mentioned above

**Comment by debo19** [comment]
> Also, have you removed all the images which are unused and is replaced by the generic image?

I've done that for most logos. Haven't checked for other images yet.

**Comment by l5io** [comment]
🚀 Preview for commit da93e85a76633b003eb4a3f64a8204cb4337c93f at: https://61ed0e16bc4aa13c28b3b327--layer5ng.netlify.app

**Comment by Nikhil-Ladha** [comment]
> > Also, have you removed all the images which are unused and is replaced by the generic image?
> 
> I've done that for most logos. Haven't checked for other images yet.

Ahh...cool. Just for the files which you have made the changes in this PR. Others van be tackled in a separate PR , if need be.

**Review by Nikhil-Ladha — APPROVED** [review]
Looks good 👌🏼

**Comment by leecalcote** [comment]
It would be good to update the CONTRIBUTING.md to include things like:

- use of `StaticImage` vs `img`.
- location of images / assets
- not to use `rel="noreferrer"` on internal hyperlinks (external only).
- use of theme colors.
- others...

**Pull request merged**